### ISSUE TITLE
Client Audit - Site Cycle Count

### ISSUE THREAD

**Issue by warrenmining** [Mining-Store/MiningStore#112]

**Comment by warrenmining** [comment]
A Cycle Count will be conducted to identify any machines which are not accounted for more in Insight, or are in Insight but not accounted for physically. This cycle count will take place once Matt Spangler has deployed the new static IP schema.

**Comment by warrenmining** [comment]
On-site is performing static ip change and cycle count. Progress can be tracked on the google sheets linked in this document under "asset management"

https://docs.google.com/document/d/1l0AjcvAZNsnI82UNomNM0Tf1Z_L3hO6iu8YEUIFTHbQ/edit?usp=sharing

**Comment by warrenmining** [comment]
I've tried to help the site solidify plans for static ip implementation. They are currently planning to do it by hand, when Foreman should be bulk-push static ip schemas using the site map. Matt has reported an error with that method - I've asked him to send an inquiry to Dan about the error but he is yet to do so. I took issue with the manual process because it would greatly increase the time to static ip, this isn't a hill I'm willing to die on.
